bchrpc/bchrpc.proto

```diff
@@ -280,6 +286,16 @@ message BlockNotification {
     BlockInfo block = 2;
 }
 
+message RawBlockNotification {
+    enum Type {
+        CONNECTED = 0;
+        DISCONNECTED = 1;
+    }
+
+    Type type = 1;
+    RawBlock block = 2;
```

Comment:
I think I would just make this `bytes` rather than nesting it in another object here.

Reply:
Yes, looks much better that way indeed. Thanks! :)